static/src/javascripts/lib/geolocation.js

```diff
@@ -10,10 +11,17 @@ const editionToGeolocationMap = {
 };
 const daysBeforeGeolocationRefresh = 10;
```

Comment:
Minor:
`daysBeforeGeolocationRefresh` is the refresh time for local storage only now.
So the name is potentially misleading.
I'd be inclined to inline the value as it looks like its only used in one place.

Reply:
Good catch!